js-exercises/add-big-integers/addBigIntegers.js

```diff
@@ -0,0 +1,43 @@
+function addBigIntegers(intString) {
+  const regexPattern = /\n/;
+  if (!regexPattern.test(intString)) {
+    throw new Error(`${intString} should has next line`);
+  }
+  const stringsList = intString.split(regexPattern);
+  const checkValidNumbers = stringsList.filter(item => Number.isNaN(parseInt(item, 10)));
```

Comment:
```suggestion
  const checkValidNumbers = stringsList.every(item => Number.isNaN(parseInt(item, 10)));
```
`every` will give you a boolean answer.